docs/integrations/databricks.md

```diff
@@ -94,7 +94,7 @@ a way to export changes from lakeFS to Unity Catalog.
 
 For the full guide on how to use Catalog Exports with Unity Catalog, see the [documentation](./unity-catalog.md).
 
-### lakeFS for Databricks <span class="badge">Cloud</span> <span class="badge">Enterprise</span>
+### lakeFS for Databricks <span class="badge">Enterprise</span>
```

Comment:
Should be Enterprise and Managed Enterprise? @ozkatz

Reply:
yes, but I think that it may be too wordy to mention both here. WDYT?